### PR TITLE
Feature/updated live view

### PR DESCRIPTION
**Pull request by cadw**
Updated the live view, it is now possible to:
- Select a camera (cameras cannot be queried by project, we should change the entity for this, I guess)
- Only query the cues from the current project
- Join the live view and see where other players are

### CODE REVIEW

**Comment by hungnguyen94** [comment]
Would it be better to keep track of the time instead of the cue? Because you can have multiple cues active at a given time. 
To filter on camera, you could add a service method to find all cues by camera. You don't need to change the entity, because the cue entity already includes the camera object.

**Comment by hungnguyen94** [comment]
Also, don't commit `index.html`.

File: src/main/java/nl/tudelft/thefirstorder/web/websocket/dto/ActivityDTO.java [hunk 1]
```diff
@@ -13,6 +13,16 @@
 
     private String page;
 
+    private String current;
+
+    public String getCurrent() {
```
Comment (ChrisBerg) [inline src/main/java/nl/tudelft/thefirstorder/web/websocket/dto/ActivityDTO.java]: Could you add javadoc comments?

File: src/main/webapp/app/admin/tracker/tracker.service.js [hunk 2]
```diff
@@ -101,24 +102,33 @@
...
             if (stompClient !== null && stompClient.connected) {
                 stompClient
                     .send('/topic/activity',
                         {},
-                        angular.toJson({'page': 'next'}));
+                        angular.toJson({'page': 'next', 'current': current}));
+            }
+        }
+
+        function sendCurrent(current) {
```
Comment (ChrisBerg) [inline src/main/webapp/app/admin/tracker/tracker.service.js]: Add javadoc comment here

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -23,216 +23,216 @@
...
 </head>
 <body ng-app="thefirstorderApp">
-    <!--[if lt IE 9]>
-        <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-    <![endif]-->
-    <div ng-show="{{ENV === 'dev'}}" class="development" ng-cloak></div>
-    <div ui-view="navbar" ng-cloak></div>
-    <div class="container">
-        <div class="well" ui-view="content"></div>
+<!--[if lt IE 9]>
```
Comment (ChrisBerg) [inline src/main/webapp/index.html]: Why did you remove the indentation?

**Comment by rubenwiersma** [comment]
Looks good, agree with Hung: what if I'm operating a camera that doesn't have the current cue? Does it stick on the cue that I'm still working on? Does it show my next cue?

**Comment by cadw** [comment]
@rubenwiersma You are right, I will try to fix this!

File: src/main/webapp/app/live/live.controller.js [hunk 4]
```diff
@@ -47,17 +55,32 @@
...
             showActivity(activity);
 
-            if (activity.page === 'next' && vm.authorized.includes(activity.ipAddress)) {
-                if (vm.current != vm.cues.length - 1) {
-                    vm.current++;
+            if (activity.current == null) {
+                JhiTrackerService.sendCurrent(0);
+                vm.current = 0;
+            } else {
+                if (activity.page === 'next' && vm.authorized.includes(activity.ipAddress)) {
```
Comment (trombomart) [inline src/main/webapp/app/live/live.controller.js]: Can you make an extra method here to decrease the conditional complexity

File: src/main/webapp/app/live/live.controller.js [hunk 5]
```diff
@@ -47,17 +55,32 @@
...
+                        JhiTrackerService.sendCurrent(vm.current);
+                        scrollToTimelineElement(true);
+                    }
                 }
-                scrollToTimelineElement(true);
-            }
-            if (activity.page === 'previous' && vm.authorized.includes(activity.ipAddress)) {
-                if (vm.current > 0) {
-                    vm.current--;
+                if (activity.page === 'previous' && vm.authorized.includes(activity.ipAddress)) {
```
Comment (trombomart) [inline src/main/webapp/app/live/live.controller.js]: Here the same

File: src/main/webapp/app/admin/tracker/tracker.service.js (hunk 2 above)
Comment (cadw) [inline src/main/webapp/app/admin/tracker/tracker.service.js]: This JavaScript :)

File: src/main/webapp/app/live/live.controller.js (hunk 5 above)
Comment (cadw) [inline src/main/webapp/app/live/live.controller.js]: I see what you mean, however, this can only be changed in a switch case. This would maybe look better, but won't change the complexity.
